app/src/main/java/com/yeonae/chamelezone/view/home/adapter/HomePlaceRvAdapter.kt

```diff
@@ -116,10 +113,15 @@ class HomePlaceRvAdapter(val currentLatitude: Double, val currentLongitude: Doub
     }
 
     fun itemChange(position: Int) {
-//        notifyItemChanged(placeList.lastIndex, 1)
         notifyItemChanged(position, 1)
     }
 
+    fun addDataDistance(lat: Double, log: Double) {
+        currentLatitude = lat
+        currentLongitude = log
+        notifyItemRangeChanged(0, itemCount, 1)
```

Comment:
```suggestion
        notifyItemRangeChanged(0, itemCount, YOUR_CONST_VAL)
```
고정된 상수로 변경하는것이 좋을 것 같아요

Reply:
수정했습니다
2a3f591